src/Extensions/Shopware/PluginCreator/template/Models/Model.tpl

```diff
@@ -1,9 +1,10 @@
 <?= $configuration->phpFileHeader; ?>
 <?= $configuration->licenseHeader; ?>
 
-namespace   Shopware\CustomModels\<?= $configuration->name; ?>;
-use         Shopware\Components\Model\ModelEntity,
-    Doctrine\ORM\Mapping AS ORM;
+namespace <?= $configuration->pluginConfig['namespace']; ?>\CustomModels\<?= $configuration->name; ?>;
```

Comment:
:+1: Thank you, looks great. There might be an issue with the CustomModels, however. Can you have a look at it?